.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Format Python and CMake code
+9fbb3c4d75ea8c2e8b49e5625e1d5f58f515539f
```

Comment:
Just a reminder that the PR needs to not get squashed/rebased/... in a way that makes this invalid.

Reply:
Thanks, indeed I was planning a merge request without squashing